test/chunking-form/samples/preserve-modules-export-alias/_expected/system/dep.js

```diff
@@ -3,7 +3,7 @@ System.register([], function (exports) {
 	return {
 		execute: function () {
 
-			const foo = exports('foo', 1);
+			const foo = function (v) { return exports({ foo: v, bar: v }); }(1);
```

Comment:
@guybedford Are you sure this is correct? If I export an object via `exports` like this, does `exports` really return `v`?

Reply:
Yes, SystemJS will return the last export value set from the function, although this is only as of 6.3.0 release on April 9. If we want to still support older SystemJS versions we can wait on this change too - it only really becomes much nicer when we can drop IE11 support and have these as arrow functions without a block.

Reply:
I see

Reply:
I think "dropping IE11", or rather dropping ES5 will not happen any time soon. Much rather I think we will have a `targetESVersion` or similar at some point, though this will be a lot more friction. There is a ton of libraries still out there that depend on this. At the very least, the story about running Babel on the output needs to be compelling and well documented before we do that.